corehq/apps/hqadmin/forms.py

```diff
@@ -51,8 +51,9 @@ def clean(self):
             if user is None:
                 raise forms.ValidationError("Cannot find user '{}'".format(username))
 
-        if not user.is_commcare_user():
-            raise forms.ValidationError("User '{}' is not a CommCareUser".format(username))
+        if domain or '.commcarehq.org' in username:
+            if not user.is_commcare_user():
+                raise forms.ValidationError("User '{}' is not a CommCareUser".format(username))
```

Comment:
I think this should use `CouchUser.get_by_username` if its expected to sometimes return `WebUser`s.

Reply:
reverted to allow login as a mobile user only, so not taking any action for this.